.buildkite/pipeline.yml

```diff
@@ -101,6 +103,36 @@ steps:
     concurrency_group: 'bitbar'
     concurrency_method: eager
 
+  - label: ':browserstack: {{matrix}} tests'
+    depends_on: "fixture"
+    timeout_in_minutes: 60
+    matrix:
+      - "ANDROID_16"
```

Comment:
It's not a big deal but you could just put ANDROID_16 in for the `--device` option and not use a matrix.  By the time Android 17 is out we _should_ have moved 16 to BB.